src/control/content/buildfire/searchTable/searchTableHelper.js

```diff
@@ -0,0 +1,239 @@
+import buildfire from 'buildfire';
+
+export class SearchTableHelper {
+	constructor(tableId, tag, config) {
+		this.supportedStores ={
+			'publicData': buildfire.publicData,
+			'userData': buildfire.userData, 
+			'appData': buildfire.appData, 
+			'datastore': buildfire.datastore
+		};
+		this.store = buildfire.datastore;
```

Comment:
Please do not change default store (publicData). This should be set in searchTableConfig.